samples/dotnet/clientcreds/readme.md

```diff
@@ -2,5 +2,52 @@
 
 Sample implementation of the Lenus Health platform demonstrating use client_credentials grant type flow to support requesting agency over a patient by a clinician within an organisation.
 
+## Running locally
```

Comment:
It might controversial, but I'd go with less detail here. With this being a c# sample I would expect that the knowledge on how to run a C# sample code is a given, including the certs. I'd maybe mention that the need to have .NET 6 SDK installed, unless it's already mentioned elsewhere